modules/rostests/apitests/browseui/IAutoComplete.cpp

```diff
@@ -493,7 +493,7 @@ DoTestCaseA(INT x, INT y, INT cx, INT cy, LPCWSTR pszInput,
     // check item count
     INT nListCount = ListView_GetItemCount(hwndList);
     if (nListCount < 1000)
-        ok_int(nListCount, nCount);
+        (void)0; // ok_int(nListCount, nCount); broken on Windows
```

Comment:
@katahiromz Could you recheck these two tests in IAutoComplete?

Reply:
Hey, yo!

WinXP:
![WinXP](https://github.com/user-attachments/assets/7c89fc2e-b360-49f4-bc2e-c44f46633380)
Successful.

Win2k3:
![Win2k3](https://github.com/user-attachments/assets/f3049454-5e31-46cc-ade6-486638c2497b)
Successful.

Win10:
![Win10](https://github.com/user-attachments/assets/173e91d5-0411-47ca-8f7c-25c4f549871d)
Successful.

LGTM from my side.

Reply:
good to know; but then, why didn't the original `ok_int(nListCount, nCount);` (now disabled in this PR) work then?